activerecord/test/cases/transactions_test.rb

```diff
@@ -541,7 +541,7 @@ def test_force_savepoint_on_instance
 
       begin
         @second.transaction requires_new: true do
-          @first.happy = false
+          @first.approved = false
```

Comment:
The assertion below is `assert_predicate @first.reload, :approved?` so we need to set the `approved` column and not the `happy` column.